conda/instructions.py

```diff
@@ -50,7 +50,7 @@ def PREFIX_CMD(state, prefix):
 def PRINT_CMD(state, arg):
     if arg.startswith(('Unlinking packages', 'Linking packages')):
         return
-    getLogger('print').info(arg)
+    getLogger('conda.stdout.verbose').info(arg)
```

Comment:
Actually, this isn't even used anymore I don't think.  Might be important to keep from backward compatible API perspective for anything that imports conda, but otherwise this whole function can just `return`.